### PR TITLE
Navigation

### PR DESCRIPTION
**Pull request by AymanGhaith**
Adding basic navigation

### CODE REVIEW

**Comment by AymanGhaith** [comment]
@JackMiszencin

**Comment by JackMiszencin** [comment]
You're going to want to put that package-lock.json file into your gitignore at some point.

File: Greenify-FrontEnd/App.js [hunk 1]
```diff
@@ -1,109 +1,25 @@
...
-    super();
-    //all the data save before sent in state
-    this.state={
-      username:'',
-      password: '',
-      phonenumber:''
-    };
+import { StyleSheet, Text, View } from 'react-native';
+import { createStackNavigator } from 'react-navigation';
+import Login from './app/components/Loggin';
```
Comment (JackMiszencin) [inline Greenify-FrontEnd/App.js:5]: make sure you spell the file name right :)

Comment (AymanGhaith) [reply Greenify-FrontEnd/App.js:5]: yes sure , thanks

**Pull request merged**